frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/explore-admin-features-getting-started-card.tsx

```diff
@@ -54,20 +40,15 @@ export const ExploreAdminFeaturesGettingStartedCard: React.FC = () => {
   );
 
   React.useEffect(() => {
-    const addLinkIfLightSpeedIsAvailable = async () => {
-      if (await getLightSpeedAvailability()) {
-        links.splice(1, 1, {
-          id: 'lightspeed',
-          title: t('public~OpenShift LightSpeed'),
-          description: t('public~Your personal AI helper.'),
-          href:
-            '/operatorhub/all-namespaces?keyword=lightspeed&details-item=lightspeed-operator-lightspeed-operator-catalog-openshift-marketplace',
-        });
-      }
-    };
-
-    addLinkIfLightSpeedIsAvailable();
-  }, [links, t]);
+    if (canListPackageManifest && canListOperatorGroup && lightspeedIsAvailable) {
+      links.splice(1, 1, {
+        id: 'lightspeed',
+        title: t('public~OpenShift LightSpeed'),
+        description: t('public~Your personal AI helper.'),
+        href: lightspeedOperatorURL,
+      });
+    }
```

Comment:
This should be in the `useMemo` hook above. Directly mutating the link array like this may not trigger a re-render of the GettingStartedCard when it changes. Something like:

```ts
  const showLightSpeedLink = canListPackageManifest && canListOperatorGroup && lightspeedIsAvailable;
  const links: GettingStartedLink[] = React.useMemo(
    () => [
      {
        id: 'openshift-ai',
        title: t('public~OpenShift AI'),
        description: t('public~Build, deploy, and manage AI-enabled applications.'),
        href:
          '/operatorhub/all-namespaces?keyword=openshift+ai&details-item=rhods-operator-redhat-operators-openshift-marketplace',
      },
      ...(
        showLightSpeedLink ? [{
          id: 'lightspeed',
          title: t('public~OpenShift LightSpeed'),
          description: t('public~Your personal AI helper.'),
          href: lightspeedOperatorURL,
        }] : []
      ),
      {
        id: 'new-translations',
        title: t('public~French and Spanish now available'),
        description: t('public~Console language options now include French and Spanish.'),
        href: '/user-preferences/language',
      },
    ],
    [t, showLightSpeedLink],
  );

Reply:
Good catch, @TheRealJon.  Addressed with https://github.com/openshift/console/pull/14116